lib/codegen/AWS_LAMBDA-response-mapping-template.tmpl.ejs

```diff
@@ -0,0 +1 @@
+$util.toJson($context.result)
```

Comment:
My response template usually looks like this

````
#if( $context.result && $context.result.errorMessage )
    $utils.error($context.result.errorMessage, $context.result.errorType, $context.result.data, $context.result.errors)
#else
    $utils.toJson($context.result.data)
#end
````

That covers the lambda returning "controlled" errors (e.g.: validation)

For example, on a createUser mutation:

success:
````js
callback(null, {
	data: {
		firstName: "John",
		lastName: "Doe",
	}
});
````

Input error:
````js
callback(null, {
	data: {},
	errorMessage: "Some Error message",
	errorType: "ValidationError",
	errors: [
		"firstName is required",
		"lastName is required",
	]
});
````